### DISCUSSION TITLE
Which filename formats do exists, like {year-locale} {month-locale} {day-locale} {page-title} ?

### DISCUSSION THREAD

**Discussion by Rollo62b** [gildas-lormeau/SingleFile#1102] [Q&A]
I'm just exploring this great tool, it  seems like can do exverything I'm looking for,
Thanks a lot for that.

I would like to have some flexibility for saving the file name, I already noticed that these options exist:
{year-utc} {month-utc} {day-utc} {year-locale} {month-locale} {day-locale} {page-title}

I usually would use a format with two-year letters, like YYMMDD.

In the FAQ I see: "_The following characters are replaced with _ in file names: ~, +, \, ?, %, *, :, |, ", <, >_"

I ask myself, are there even more hidden format options in this tool or ca nthey easily added, which could allow even more flexible name formatting ?
Is there a complete list of all formats somewhere in the description available ( I couldn't find it ) ?

**Comment by gildas-lormeau** [comment]
The detailed documentation is only in the help page of the extension currently, see below if you can read HTML :).

https://github.com/gildas-lormeau/SingleFile/blob/e49cfe30bf9b37a828512aaf018c1774f4c2131f/src/ui/pages/help.html#L721-L824

Getting `YY` will be complicated, however you can format it into `YYYYMMDD` by using `{year-utc}{month-utc}{day-utc}`.

Hidden options are documented here: https://github.com/gildas-lormeau/SingleFile/wiki/Hidden-options.
